subworkflows/local/hic_mapping.nf

```diff
@@ -210,32 +210,37 @@ workflow HIC_MAPPING {
     // ch_versions         = ch_versions.mix ( SNAPSHOT_HRES.out.versions )
 
     //
-    // MODULE: MERGE POSITION SORTED BAM FILES AND MARK DUPLICATES
+    // MODULE: SUBSAMPLE BAM IF OVER 50G
```

Comment:
Would it be better to check file size and use an if block if size over 50G prior to the module. This would then stop a container being spun up just to `mv` a file. Then the module can focus on subsampling.

Something akin to:
```
file = input.map{ it -> it[1] }
if ( file.size() > 50G ) {
    SUB_SAMPLE(input)
    fixed_ch = SUB_SAMPLE.out.subsampled
} else {
    fixed_ch = input
}
```